packages/babel-helper-validator-identifier/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@babel/helper-validator-identifier",
+  "version": "7.9.0",
+  "description": "Validate identifier/keywords name",
+  "repository": "https://github.com/babel/babel/tree/master/packages/babel-helper-validator-identifier",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "./lib/index.js",
+  "exports": "./lib/index.js",
```

Comment:
Following practice from https://github.com/nodejs/node/pull/32098/files#diff-8e67f407bc32a0569e25d7ecaff6e494R208

`"exports": string` is supported on every version of node.js 13.